src/KubeOps/Operator/Controller/ManagedResourceController{TEntity}.cs

```diff
@@ -128,6 +130,13 @@ protected async Task HandleResourceEvent(ResourceEvent<TEntity> resourceEvent)
                         return;
                 }
             }
+            catch (HttpOperationException hoe) when (hoe.Response.StatusCode == HttpStatusCode.UnprocessableEntity)
+            {
+                var status = KubernetesJson.Deserialize<V1Status>(hoe.Response.Content);
+                _logger.LogWarning(status.Message);
```

Comment:
Only one small change: I would like to have some context and not just the error message:
```suggestion
                _logger.LogWarning(@"Reconciling ""{eventType}"" on resource ""{kind}/{name}"" resulted in an unprocessable entity: {errorMessage}",
                eventType,
                resource.Kind,
                resource.Name(),
                status.Message);
```

Reply:
Thanks for the feedback. I did that because the Requeue Error message logs in that exact format. So this Warning message would return the preformatted, multi-line error message from the API Server followed by that format you were recommending. Furthermore, the Warning level is usually excluded for this namespace but can be turned on to get additional information. 

`protected void RequeueError(ResourceEvent<TEntity> resourceEvent, Exception ex)
    {
        _metrics.ErroredEvents.Inc();

        var attempt = resourceEvent.Attempt + 1;
        var delay = _settings.ErrorBackoffStrategy(attempt);

        _logger.LogError(
            ex,
            @"Event type ""{eventType}"" on resource ""{kind}/{name}"" threw an error. Retry attempt {retryAttempt}.",
            resourceEvent.Type,
            resourceEvent.Resource.Kind,
            resourceEvent.Resource.Name(),
            attempt);

        _eventQueue.EnqueueLocal(resourceEvent with { Attempt = attempt, Delay = delay });
    }`